index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <title>Portfolio - Mobile Version</title>
+    </head>
+    <body>
+        <main>
+            <div class="header-section">
+                <header class="header">
+                    <nav class="nav">
+                        <img src="assets/my-logo.png" alt="My Logo" />
```

Comment:
- [ ] Your logo will be better with a link (`a` tag). With the link, you can easily redirect users to the homepage.